CMakeLists.txt

```diff
@@ -24,10 +24,21 @@ if(NOT "${CMAKE_BUILD_TYPE}" MATCHES "^(Debug|Release|RelWithDebInfo|MinSizeRel)
   message(FATAL_ERROR "Please re-run CMake, specifying -DCMAKE_BUILD_TYPE=Debug , -DCMAKE_BUILD_TYPE=Release , -DCMAKE_BUILD_TYPE=RelWithDebInfo or -DCMAKE_BUILD_TYPE=MinSizeRel .")
 endif()
 
-option(BUILD_SHARED_LIBS "Build shared library" ON)
+option(FRUIT_ALLOW_MULTI_CONFIG "Allow multi-configuration generator. If this option is OFF, it enforces the same behavior as single-configuration generator." OFF)
```

Comment:
"enforces the same behavior as single-configuration generator" is a bit vague. How about "enforces that a single configuration is used" or similar?

Reply:
Update comments dc4ee1e4df4eaf7fa459e6b2efdd3425f51d2af0